CondFormats/SiPhase2TrackerObjects/src/TrackerDetToDTCELinkCablingMap.cc

```diff
@@ -82,6 +82,18 @@ std::vector<uint32_t> TrackerDetToDTCELinkCablingMap::getKnownDetIds() const {
   return knownDetId;
 }
 
+std::vector<uint32_t> TrackerDetToDTCELinkCablingMap::getAllDetIdsForDTCId(unsigned int dtcId) const {
+  std::vector<uint32_t> result;
```

Comment:
This is being called every event. Considering storing it as a data member of TrackerDetToDTCLinkCablingMap, so it only needs storing once. (I think the other cabling map class, which we decided not to use contains tricks to distiguish data members that are part of the DB object from data members that are not).